### PR TITLE
created account table, and profile table. Also added all future tables to admin.

### PR DESCRIPTION
**Pull request by peppacaiou**
#124 
#125 
#127 

image
image

### CODE REVIEW

**Review by sosropartono — APPROVED** [review]
Looks good to me!

**Comment by K0725** [comment]
It looks good. It seems like your database interact with django administration as well.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (rezatnoMsirhC) [inline .vscode/settings.json]: Please add `.vscode` to the `.gitignore` since this is an IDE specific file and shouldn't be required for everyone

File: Mesh.session.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DROp TAble app_accounts
```
Comment (rezatnoMsirhC) [inline Mesh.session.sql:1]: Why is this required?

File: mesh/accounts/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.2 on 2023-06-14 18:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
```
Comment (rezatnoMsirhC) [inline mesh/accounts/migrations/0001_initial.py:8]: What's this for? Leftover from development/debugging?

File: mesh/conversation/models.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
 from django.db import models
 
 # Create your models here.
+
+class Message(models.Model):
+    pass
```
Comment (rezatnoMsirhC) [inline mesh/conversation/models.py:6]: Was this left here by accident, or is it necessary and will be expanded on later? Having a `TODO` here would help clear things up.

File: .vscode/settings.json
Comment (Leaversa) [inline .vscode/settings.json:1]: Should gitignore this

File: mesh/accounts/migrations/0001_initial.py
Comment (Leaversa) [inline mesh/accounts/migrations/0001_initial.py:1]: Not sure if its intentional but displayTheme, connnectedAccounts, and enable2Factor is missing

File: mesh/accounts/models.py
Comment (Leaversa) [inline mesh/accounts/models.py:1]: Again displayTheme, connnectedAccounts, and enable2Factor is missing

File: mesh/accounts/views.py
Comment (Leaversa) [inline mesh/accounts/views.py:1]: Is this just for testing?

File: Mesh.session.sql (hunk 2 above)
Comment (peppacaiou) [reply Mesh.session.sql:1]: this wasa debug file i used, i will be dropping this

File: mesh/accounts/migrations/0001_initial.py (hunk 3 above)
Comment (peppacaiou) [reply mesh/accounts/migrations/0001_initial.py:8]: It aint broke so i aint fix

File: mesh/conversation/models.py (hunk 4 above)
Comment (peppacaiou) [reply mesh/conversation/models.py:6]: This was a template for future people

File: mesh/profiles/migrations/0001_initial.py
Comment (Leaversa) [inline mesh/profiles/migrations/0001_initial.py:1]: minor suggestion but instead of image put profilePicture since its a bit more clear

**Review by Leaversa — CHANGES_REQUESTED** [review]
needs some corrections

**Review by Leaversa — APPROVED** [review]
lgtm

File: mesh/accounts/migrations/0001_initial.py (hunk 3 above)
Comment (rezatnoMsirhC) [reply mesh/accounts/migrations/0001_initial.py:8]: I feel that, but you should understand your code so we can reduce unnecessary bloat, fix things more easily when shit breaks, etc.

**Comment by misslame** [comment]
Reverted this because it did not follow Commit convention :)

**Comment by misslame** [comment]
Ugh Nevermind, I can't revert it. This will just be a stain on our commit history. @peppacaiou , FOLLOW COMMIT CONVENTIONS: 

" #number A#.# What you did"
